src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -141,7 +141,9 @@ describe('Step Image output', () => {
     ));
 
     // select aws as upload destination
-    await user.click(await screen.findByTestId('upload-aws'));
+    await waitFor(
```

Comment:
`waitFor()` blocks can be also completely removed from this test. I'd personally prefer that as I feel they only clutter up the async logic. In this case the removal would be a bit more involved as there are other user actions that would need to be awaited.

As this is not the main focus of this PR, I'd say let's go with the `waitFor()` blocks for now and I'd later clean them up a bit. Would that work?